draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -12663,13 +12663,12 @@ Content-Type: text/plain
 
 <reference anchor="HTTP2">
```

Comment:
Above the `<references>` section, you should add:

`<displayreference target="HTTP2" to="HTTP/2"/>`